### ISSUE TITLE
Integrate data from eskom calendar

### ISSUE THREAD

**Issue by deavmi** [thehta/loadshedding-announcer#1]
You could get proper data from Boyd Kane's eskom calendar and revive this epic project!

https://github.com/beyarkay/eskom-calendar

**Comment by thehta** [comment]
@deavmi  Yes, good idea. I am looking at using Boyd's calendar to set up a push notification server with ntfy.sh.

**Comment by deavmi** [comment]
That's a neat way to do it!

I actually mentioned ntfy.sh to @Boyd Kane
